src/ompl/base/objectives/src/MaximizeMinClearanceObjective.cpp

```diff
@@ -50,9 +50,14 @@ ompl::base::Cost ompl::base::MaximizeMinClearanceObjective::stateCost(const Stat
     return Cost(si_->getStateValidityChecker()->clearance(s));
 }
 
+bool ompl::base::MaximizeMinClearanceObjective::isSatisfied(Cost c) const
+{
+    return isCostBetterThan(c, Cost(threshold_.value() + magic::BETTER_PATH_COST_MARGIN));
```

Comment:
This is consistent to what it currently is.

In both cases (`MaximizeMinClearanceObjective` and the default `OptimizationObjective`) the effect of the MARGIN is to make the threshold tougher to satisfy. Whether that's how it *should* be depends on what the original motivation was...